src/utils/utils.ts

```diff
@@ -248,6 +232,20 @@ export type TriggerSettings<
   };
 };
 
+export type QueryMembersFunction<
```

Comment:
Not sure if we have some pattern in this file. So just keeping this type close to where its used!

Reply:
Maybe just more it next to the function itself in the file? Types are not all together in there anyway.